models/post-comment-model.js

```diff
@@ -0,0 +1,43 @@
+//REQUIRES
+const db = require("../db/connection.js");
+
+//FUNCTION
+function newComment(article_id, postingComment) {
+    let dateObj = new Date();
```

Comment:
You don't need to add a `Date` value, SQL will add that property on for you so it can be omitted from the query 👍